tests/Endpoints/IndexTest.php

```diff
@@ -192,7 +192,10 @@ public function testGetTasks(): void
         $results = array_unique($allIndexUids);
         $expected = [$this->index->getUid(), 'other-index'];
 
-        $this->assertSame(sort($results), sort($expected));
+        sort($results);
+        sort($expected);
+
+        $this->assertSame($results, $expected);
```

Comment:
`sort` always returns true, so phpstan complained about that: `Call to method PHPUnit\Framework\Assert::assertSame() with true and true will always evaluate to true.`

Reply:
And voila, now tests fails because the expectations were wrong :D
```
1) Tests\Endpoints\IndexTest::testGetTasks
Failed asserting that two arrays are identical.
--- Expected
+++ Actual
@@ @@
 Array &0 (
     0 => 'index_[29](https://github.com/meilisearch/meilisearch-php/actions/runs/3946584981/jobs/6754483260#step:7:30)2b37af8be3c1639bd46d3d8b08a16f'
+    1 => 'other-index'
 )
```

Reply:
Not sure what this code is supposed to test so waiting for feedback

Reply:
Hi @norkunas thanks for pointing this out.

The idea was to test the ability to compose a `getTasks` request from an index instance where I could also use the value defined inside the instance.

In this case, we want to get all the tasks from both `$this->index` and `other-index`. And we just want to ensure the presence of both indexUids in the response. They don't need to respect the order in the array btw.

When I implemented this test I didn't realize that `sort()` affects the value instead of creating a new array 🤦.

Reply:
There was a bug :)